src/test/datascience/interpreters/environmentActivationService.vscode.test.ts

```diff
@@ -52,13 +55,23 @@ suite('DataScience - VSCode Notebook - (Conda Execution) (slow)', function () {
             api = await initialize();
             sinon.restore();
             pythonApiProvider = api.serviceContainer.get<IPythonApiProvider>(IPythonApiProvider);
-            const interpreter = await api.serviceContainer
-                .get<IInterpreterService>(IInterpreterService)
-                .getActiveInterpreter();
-            if (!interpreter) {
+            const interpreterService = api.serviceContainer.get<IInterpreterService>(IInterpreterService);
```

Comment:
Improved the test to work locally more often. The active interpreter isn't always a conda one locally. This should ensure it's at least picking a conda one.